### ISSUE TITLE
Decision Document Created for API Architecture

### ISSUE THREAD

**Issue by indytechcook** [department-of-veterans-affairs/va.gov-cms#6501]
## Description

To help make the decision of what the API architecture should looks like, a decision document should be created.   The format should follow the AP - Solution Options Matrix.

The criteria should take into consideration the output from https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6490 and https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/public-content-api/architecture.md

The criteria should take into consideration technical implementation as well as maintenance.

## Acceptance Criteria
- [ ] Create Google Spreadsheet with Criteria
- [ ] Come up with evaluation Criteria
- [ ] Review Criteria with team Platform CMS Team
- [ ] PO has Approved Criteria
- [ ] Fill in document with findings from https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6491 and https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6666

**Comment by jefflbrauer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VAgov-CMS-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/6501?planning-poker) with ZenHub @indytechcook @timcosgrove @ElijahLynn @ndouglas @olivereri

**Comment by ElijahLynn** [comment]
tldr; decision on using a Node layer in between Drupal or not.
